android/libraries/rib-workflow-test/src/main/java/com/uber/rib/workflow/core/StepTester.java

```diff
@@ -13,13 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
We could consider adding Spotless to make sure we have consistent spacing and formatting.

Reply:
Yeah, totally makes sense & and I'd like to do that once done with this, but for now, I just want the internal & external versions to match exactly (since internal doesn't run Spotless) so that the remaining differences are more clearly visible.